### DISCUSSION TITLE
Simple Query - Unable to run the fargate command using GitHub Action with --launch-config flag

### DISCUSSION THREAD

**Discussion by VikasKrishnamraju** [artilleryio/artillery#3400] [Artillery]
Hi a simple question, unable to run the fargate command while using the -launch-config flag and passing the json string.
Tried to escape lot ways but many failed attemps.. Not sure what im missing to escape in yml or to include anything..

git hub actions yml file : https://github.com/VikasKrishnamraju/artillery_poc/blob/main/.github/workflows/ecs_artillery_actions.yml
<img width="1326" alt="image" src="https://github.com/user-attachments/assets/d5b1c35c-b4ce-49f2-a3a1-858b4c05ca2a">

Below is the error encountered.
<img width="1278" alt="image" src="https://github.com/user-attachments/assets/cb3d81e1-40e6-4abb-92c3-747321343400">

**Comment by hassy** [comment]
It's a YAML quoting issue. For your specific use, you can use --cpu and --memory flags instead

**Reply by VikasKrishnamraju** [reply]
Thanks @hassy that worked. Gone through the documentation once again now its clear
